diff -ru node_modules.orig/request/node_modules/form-data/node_modules/combined-stream/lib/combined_stream.js node_modules/request/node_modules/form-data/node_modules/combined-stream/lib/combined_stream.js
--- node_modules.orig/request/node_modules/form-data/node_modules/combined-stream/lib/combined_stream.js	2013-01-21 05:43:52.000000000 +0000
+++ node_modules/request/node_modules/form-data/node_modules/combined-stream/lib/combined_stream.js	2013-01-21 05:44:40.000000000 +0000
@@ -67,7 +67,7 @@
   var stream = this._streams.shift();
 
 
-  if (!stream) {
+  if (typeof stream=="undefined") {
     this.end();
     return;
   }
